pyannote/audio/labeling/models.py

```diff
@@ -53,15 +53,20 @@ class StackedLSTM(object):
         Defaults to [16, ] (i.e. one dense MLP layers with 16 units)
     n_classes : int, optional
         Number of output classes. Defaults to 2 (binary classification).
+    activation_output : {'softmax', 'sigmoid'}, optional
+        Activation for output layer. Default to 'softmax'. For binary 
+        classification, if we want use 'sigmoid' activation function,
+        the n_classes should be set to 1.
```

Comment:
Hmmmm. I agree that it does not make much sense to use `sigmoid` for the final activation if the output dimension is higher than 1 -- but it won't break, right?